src/writers/writer.ts

```diff
@@ -16,6 +16,7 @@ export interface IWriteOptions {
   xAlign: string;
   yAlign: string;
   textRotation: number;
+  textShear?: number;
```

Comment:
Also rotating is different than shearing because in shearing the lines don't align with eachother, whereas text rotation will keep all lines aligned.